Sources/Redis/Application+Redis.swift

```diff
@@ -3,7 +3,7 @@ import Vapor
 extension Application {
     public struct Redis {
         func pool(for eventLoop: EventLoop) -> RedisConnectionPool {
-            self.application.redisStorage.pool(for: self.eventLoop.next(), id: self.id)
+            self.application.redisStorage.pool(for: eventLoop, id: self.id)
```

Comment:
I'd like to dig more into why this is broken now vs. before - I did a diff comparison and this is how it worked before #178 - but if unit tests pass and people are reporting it fixes it.. then I guess we can merge. Worst case is we're in the same situation

Reply:
Yeah, I made sure to preserve the existing logic. Glad this change helps!